bundles/org.openhab.binding.herzborg/README.md

```diff
@@ -0,0 +1,81 @@
+# Herzborg Binding
+
+This binding supports smart curtain motors by Herzborg (http://www.herzborg.com/pro_list.aspx?TypeID=1)
+
+## Supported Things
+
+- `herzborg` A bridge thing that connects to a RS485 serial bus.
+- `curtain` A curtain motor thing that can be controlled via the `herzborg` bridge .
+
+The binding was developed and tested using DT300TV-1.2/14 type motor; others are expected to be compatible
+
+## Discovery
+
+Due to nature of serial bus being used, no automatic discovery is possible.
+
+## Thing Configuration
+
+### Serial Bus Bridge (id "serial_bus")
+
+| Parameter | Meaning                                                 |
+|-----------|---------------------------------------------------------|
+| port      | Serial port name to use                                 |
+
+Herzborg devices appear to use fixed 9600 8n1 communication parameters, so no other parameters are needed
+
+### Curtain Motor Thing (id "curtain")
+
+| Parameter     | Meaning                                                 |
+|---------------|---------------------------------------------------------|
+| address       | Address of the motor on the serial bus.                 |
+| poll_interval | Polling interval in seconds                             |
+
+## Channels
+
+| channel    | type          | description                                   |
+|------------|---------------|-----------------------------------------------|
+| position   | RollerShutter | Controls position of the curtain. Position reported back is in percents; 0 - fully closed; 100 - fully open |
+| mode       | String        | Reports current motor mode:                   |
+|            |               | 0 - Stop                                      |
+|            |               | 1 - Open                                      |
+|            |               | 2 - Close                                     |
+|            |               | 3 - Setting                                   |
+| reverse    | Switch        | Reverses direction when switched on           |
```

Comment:
In fact it is a state. It can be written, it's stored in the device and it can be read back.
What it basically does is reversing meaning of 0% and 100%. By default 100% is fully closed and 0% is fully open. Setting "reverse" to true swaps that. And yes, i know that openhab actually prefers it reversed.
I agree that this is not end-user setting, it's purely technical, used only during setup. But the hardware comes with no software tools, only with protocol description, so i decided to make all the settings available, just in case. There aren't many of them, so it was easy.

Reply:
OK